src/diffusers/models/unets/unet_spatio_temporal_condition.py

```diff
@@ -397,6 +397,8 @@ def forward(
 
         # broadcast to batch dimension in a way that's compatible with ONNX/Core ML
         batch_size, num_frames = sample.shape[:2]
+        if torch.is_tensor(num_frames):
```

Comment:
How can `num_frames` ever be a tensor? Can you give an example?

Reply:
@patrickvonplaten `num_frames` is created as a CPU tensor during the tracing step of the ONNX export. I have also provided a script to reproduce this behavior in the [comment ](https://github.com/huggingface/diffusers/pull/6562#issuecomment-1972226200) below